server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -682,6 +682,9 @@ public void setKubernetesServiceHelpers(final List<KubernetesServiceHelper> kube
             HypervisorType.XenServer,
             HypervisorType.Simulator
     ));
+    public static final List<HypervisorType> VM_STORAGE_MIGRATION_WITH_SUPPORTING_SUPPORTING_HYPERVISORS = new ArrayList<>(Arrays.asList(
```

Comment:
the name is a bit confusing

Reply:
Thanks @weizhouapache. Too many `SUPPORTING`. Fixed now